cl/corpus_importer/tasks.py

```diff
@@ -1794,6 +1796,10 @@ def get_att_report_by_rd(
     pacer_court_id = map_cl_to_pacer_id(rd.docket_entry.docket.court_id)
     att_report = AttachmentPage(pacer_court_id, s)
     att_report.query(rd.pacer_doc_id)
+    if is_appellate_court(pacer_court_id):
+        att_report = AppellateAttachmentPage(pacer_court_id, s)
+    else:
+        att_report = AttachmentPage(pacer_court_id, s)
```

Comment:
While testing this with a real Fetch request, I noticed it was failing at this part:

It worked after applying this tweak:
```suggestion
    if is_appellate_court(pacer_court_id):
        att_report = AppellateAttachmentPage(pacer_court_id, s)
    else:
        att_report = AttachmentPage(pacer_court_id, s)
    att_report.query(rd.pacer_doc_id)
```

Reply:
You're correct. Thanks!